packages/grafana-data/src/geo/layer.ts

```diff
@@ -49,7 +49,7 @@ export interface FrameGeometrySource {
  */
 export interface MapLayerOptions<TConfig = any> {
   type: string;
-  name?: string; // configured display name
+  name?: string; // configured unique display name
```

Comment:
in a follow up PR, lets make this required.  Once we guarantee it is unique and always set we can get rid of UID also